gdk/commands/component/ListCommand.py

```diff
@@ -25,27 +27,36 @@ def run(self):
             return
         raise Exception(error_messages.LIST_WITH_INVALID_ARGS)
 
-    def get_component_list_from_github(self, url):
-        comp_list_response = requests.get(url)
-        if comp_list_response.status_code != 200:
```

Comment:
could you add some explanation here for why removing check of 200?

Reply:
That is what raise_for_status() already does